src/status_im/contexts/wallet/common/activity_tab/view.cljs

```diff
@@ -31,11 +31,21 @@
         on-end-reached (rn/use-callback
                         #(rf/dispatch
                           [:wallet/get-more-for-activities-filter-session]))]
-    (if (and (and (some? loading?) (false? loading?)) (empty? activity-list))
+    (cond
+      (and (empty? activity-list) loading?)
+      [rn/view
+       [quo/skeleton-list
+        {:content       :messages
+         :parent-height (:height (rn/get-window))
+         :animated?     false}]]
```

Comment:
Do you know if we need to wrap it in view? if not, we can use the skeleton directly. WDYT?